### PR TITLE
Nomos update 18-12-2023

### PR DESCRIPTION
**Pull request by Cofson**

### CODE REVIEW

File: content/nomos/updates/2023-12-18.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+draft: false
+description: Weekly update of Nomos
+---
+## `network privacy and mixnet`
+
+### `research`
+
+- Continuing the research on the effects of wealth concentration within Ouroboros Crypsinous - more precisely fixing some of the code and running simulations. The results and reports will continue to be provided in the [Notion doc](https://www.notion.so/Does-Crypsinous-Leader-Election-Function-lead-to-wealth-concentration-in-PoS-b81f07a791b745438443f51f00ac258f).
+- Also, doing additional research into the Darkfi's Crypsinous implementation (and why they are striving away from it)
+- Solved [an issue](https://github.com/logos-co/nomos-node/pull/544](https://github.com/logos-co/nomos-node/pull/544 "https://github.com/logos-co/nomos-node/pull/544)  that was causing problems with simulations (Carnot 10k nodes - Tree implementation returned different results when searching for child committee and then using child committee to find its parent)
```
Comment (alvatar) [inline content/nomos/updates/2023-12-18.md]: Why do you think this is research @Cofson ? Being code, doesn't it seem like it fits better the development section?

File: content/nomos/updates/2023-12-18.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- By fixing the above and the depth calculation in the analysis script, the simulations stall  has been fixed totally, and we are meeting in the deadlines in terms of releasing the Carnot papers - no additional showstoppers
```
Comment (alvatar) [inline content/nomos/updates/2023-12-18.md]: likewise

File: content/nomos/updates/2023-12-18.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- [Mixnet specification](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2) is done, currently in review - once additional comments have been added, will provide new updates
+- There have been some slight improvements in the [Crypsinous over Mixnet document](https://www.notion.so/Crypsinous-over-Mixnet-dc05e9ad601a4cdfb1ac7823fd51cff6) - not too major
+- Compiled a [high-level overview](https://www.notion.so/Ouroboros-Family-8cd7285459f640b088716905ab03474f) of the Ouroboros family
+- Studying [byzantine gossip and swarm consensus](https://www.notion.so/Byzantine-Gossip-Consensus-a83513712031444f80eb5f6097773b72) as part of the effort of solving the tagging attack problem (as alternatives to random subsampling and reliable broadcast) - some of the ideas involve the use of gradecast over byzantine gossip, but providing more details in the following week.
+- The architecture whitepaper is awaiting feedback and expected to be finalized in the first week of 2024
```
Comment (alvatar) [inline content/nomos/updates/2023-12-18.md]: The architecture whitepaper is not a Mixnet document...

File: content/nomos/updates/2023-12-18.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- We’ve run simulations of the process and they confirm the stability conditions and convergence derived analytically (above)
+- There is still work to be done in terms of slots observation to understand a good measurement of leader rate as well as some additional confirmation of the analysis
+- Analysis of stake de-anonymization (details are [here](https://www.overleaf.com/read/xvgmfchdhgzh#acd15d](https://www.overleaf.com/read/xvgmfchdhgzh#acd15d "https://www.overleaf.com/read/xvgmfchdhgzh#acd15d) and here): and [here](https://www.overleaf.com/read/fzbrxvkwwscq#f2907c))
+- Used the lower bound (LB), asymptotic estimate (AE) and upper bound (UB) on the probability of stake greater than some threshold ( “confidence”), within a given “accuracy”, to estimate the number of layers in the mixnet.
```
Comment (alvatar) [inline content/nomos/updates/2023-12-18.md]: correct ( "confidence") -> ("confidence")

**Review by alvatar — CHANGES_REQUESTED** [review]

File: content/nomos/updates/2023-12-18.md (hunk 1 above)
Comment (Cofson) [reply content/nomos/updates/2023-12-18.md]: that was a mistake indeed - migrated to development

File: content/nomos/updates/2023-12-18.md (hunk 2 above)
Comment (Cofson) [reply content/nomos/updates/2023-12-18.md]: same as above - migrated to development, thanks

File: content/nomos/updates/2023-12-18.md (hunk 4 above)
Comment (Cofson) [reply content/nomos/updates/2023-12-18.md]: fixed, thanks

**Review by alvatar — APPROVED** [review]

**Pull request merged**